scripts/generate-entry-i18n-chunks.js

```diff
@@ -0,0 +1,222 @@
+// cli usage
```

Comment:
@paninee do we need this? it looks like it is a duplicate for `scripts/generate-entry-i18n.js`